prime-router/src/main/kotlin/transport/SftpTransport.kt

```diff
@@ -4,42 +4,44 @@ import com.jcraft.jsch.ChannelSftp
 import com.jcraft.jsch.JSch
 import com.jcraft.jsch.Session
 import gov.cdc.prime.router.OrganizationService
+import gov.cdc.prime.router.azure.ReportQueue
 import sftputils.SftpUtils
 import java.util.*
 
-class SftpTransport {
+class SftpTransport : Transport{
 
-    fun send(transport: OrganizationService.Transport, contents: ByteArray, fileName: String) {
+    override fun send(service: OrganizationService, header: ReportQueue.Header, contents: ByteArray) : Boolean {
 
-        val session = initSshClient(transport.host, transport.port)
-        val fileDir = "/sftpout"
-        val path = "${fileDir}/${fileName}.csv"
+        val fileDir = "./upload"
+        val path = "${fileDir}/${service.fullName.replace( '.', '-')}-${header.id}.csv"
+        val host: String = service.transport.host 
+        val port: String = service.transport.port
+        val user: String = "foo"
+        val password: String = "pass"
 
-        val channel: ChannelSftp = session.openChannel("stfp") as ChannelSftp
+        val jsch = JSch()
+        val jschSession = jsch.getSession(user, host, port.toInt() )
+        val config = Properties(); 
+        config.put("StrictHostKeyChecking", "no")
+        config.put("PreferredAuthentications", "password");
+        jschSession.setConfig(config)
+        jschSession.setPassword(password)
+        
+        jschSession.connect(5000)
```

Comment:
can we get a constant here instead of a magic number? I don't quite know what this is doing by reading it.

Reply:
This is the default value for connect -- so I just removed